README.md

```diff
@@ -15,6 +17,41 @@ def deps do
 end
 ```
 
+## Configuration
+
+1. Configure StathamLogger in `config.exs`. In this example `password` and `just_another_sensitive_key` will have corresponding value set to `[FILTERED]:
```

Comment:
Replace `config.exs` with conig. It is OK to configure logger in `runtime.exs`.

Reply:
done